substrate/frame/staking/src/lib.rs

```diff
@@ -387,7 +388,76 @@ impl<AccountId: Ord> Default for EraRewardPoints<AccountId> {
 	}
 }
 
+/// The payout destination for an account.
+///
+/// NOTE: Being lazily migrated to. Logic pertaining to this enum has been introduced to `set_payee`
+/// and payout logic, replacing `RewardDestination`.
+#[derive(PartialEq, Eq, Copy, Clone, Encode, Decode, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub enum PayoutDestination<AccountId> {
+	/// Payout goes into the stash account and is added to bond.
+	Stake,
+	/// Deposit the specified percentage of payout to the specified account as free balance, and
+	/// pay the rest into the stash account and add to bond. 0% and 100% should be disallowed and
+	/// handled as `Stake` and `Deposit` respectively.
+	Split((Perbill, AccountId)),
+	/// Deposit payout as free balance to an account.
+	Deposit(AccountId),
+	/// Receive no payout.
+	Forgo,
+}
+
+// Used for testing and benchmarking where stash and controller accounts are sometimes generated
```

Comment:
nit but if this is only used for testing and benches, maybe we could have conditional compilation attribute here?

Reply:
It seems because we are implementing `PayoutRoute` in this file, it needs to be in the scope, so `#[cfg(any(test, feature = "runtime-benchmarks"))]` will result in an error.